microsoft-edge/devtools-guide-chromium/javascript/source-maps.md

```diff
@@ -82,21 +82,19 @@ Source Maps are enabled by default.<!-- (as of Microsoft Edge 39)-->
 
 To make sure that Source Maps are enabled:
 
-1. Open DevTools.
+1. In DevTools, click **Settings** (![Settings icon.](../media/settings-gear-icon-light-theme.png)) > **Preferences**.
```

Comment:
A critique of the original breakout of steps: a big antipattern in the upstream original docs is unnecessary dependency on hyperlinking away from the present page merely to present one viable way to get the job done, of merely opening DevTools.  We're instead inlining that; if we keep the separate step about first opening DevTools, we must present, right there, one viable way to accomplish opening DevTools, not link away as if it is so complicated, you have to abandon the present article and get lost in hypertext space.  In your revision, you silently assume that DevTools is already open.  There's no rule established about whether to have a Step 1: OpenDevTools by doing X, but keep in mind that in some cases, "Every page is Page 1" -- recently we had an Issue logged complaining that the docs simply assume familiarity, but the reality is that this user came into the page from a web search, they explicitly pointed out.  Consider: Don't alienate users by simply assuming that they know that they need to open DevTools, and how to accomplish that.  It doesn't matter at all which way we provide, inline, to open DevTools, but (if we choose to doc that in a given h2 section's action-steps), we need to state at least one way to get the job done.